kaolin/csrc/cuda/ops/conversions/unbatched_mcube/unbatched_mcube_cuda_kernel.cu

```diff
@@ -491,28 +495,52 @@ generateTriangles2(float *pos, int *faces, int *compactedVoxelArray,
     num_prev_verts = numPartialVertsScanned[target_voxel_idx3];
     num_prev_triangles = numTrianglesScanned[voxel];
 
-    faces[num_prev_triangles * 3 + j + 2] = num_prev_verts + offset3;
-  }
+    faces[num_prev_triangles * 3 + j] = num_prev_verts + offset3;
+    }
 }
 
 void launch_generateTriangles2(at::Tensor pos, at::Tensor faces, at::Tensor compactedVoxelArray,
-                              at::Tensor numTrianglesScanned, at::Tensor numPartialVertsScanned, at::Tensor numPartialVerts,
-                              at::Tensor voxelVertsOrder,
-                              int3 gridSize, at::Tensor voxelgrid,
-                              float3 voxelSize, float isoValue, int activeVoxels, int maxVerts)
+                                at::Tensor numTrianglesScanned, at::Tensor numPartialVertsScanned, at::Tensor numPartialVerts,
+                                at::Tensor voxelVertsOrder,
+                                int3 gridSize, at::Tensor voxelgrid,
+                                float3 voxelSize, float isoValue, int activeVoxels, int maxVerts)
 {
-  dim3 grid2((int) ceil(activeVoxels/ (float) NTHREADS), 1, 1);
+    dim3 grid2((int) ceil(activeVoxels/ (float) NTHREADS), 1, 1);
 
-  while (grid2.x > 65535) {
+    while (grid2.x > 65535) {
     grid2.x/=2;
     grid2.y*=2;
-  }
-
-  generateTriangles2<<<grid2, NTHREADS>>>(pos.data_ptr<float>(), faces.data_ptr<int>(),
-                                          compactedVoxelArray.data_ptr<int>(), numTrianglesScanned.data_ptr<int>(),
-                                          numPartialVertsScanned.data_ptr<int>(), numPartialVerts.data_ptr<int>(),
-                                          voxelVertsOrder.data_ptr<int>(),
-                                          voxelgrid.data_ptr<float>(), gridSize,
-                                          voxelSize, isoValue, activeVoxels,
-                                          maxVerts);
+    }
+
+    generateTriangles2<<<grid2, NTHREADS>>>(pos.data_ptr<float>(), faces.data_ptr<int>(),
+                                            compactedVoxelArray.data_ptr<int>(), numTrianglesScanned.data_ptr<int>(),
+                                            numPartialVertsScanned.data_ptr<int>(), numPartialVerts.data_ptr<int>(),
+                                            voxelVertsOrder.data_ptr<int>(),
+                                            voxelgrid.data_ptr<float>(), gridSize,
+                                            voxelSize, isoValue, activeVoxels,
+                                            maxVerts);
```

Comment:
Not sure why the spacing changed (auto-indentation with your IDE ?) with C++ we usually follow [the Google style guide](https://google.github.io/styleguide/cppguide.html) , which means 2 space for this kind of indentation. Although we've losely enforced it, I prefer avoid any modification that would go against it and make the review harder.

Reply:
Maybe it's a copy-paste issue, fixed.